Remove bundled dependencies.

We have to keep minizip for now because the newest API is not backward
compatible with the core codebase.
--- a/Makefile.common
+++ b/Makefile.common
@@ -1,22 +1,22 @@
 LIBRETRO_DEPS_DIR    = $(CORE_DIR)/deps
-LIBOGG_DIR           = $(LIBRETRO_DEPS_DIR)/libogg
-LIBVORBIS_DIR        = $(LIBRETRO_DEPS_DIR)/libvorbis
 LIBCHDR_DIR          = $(LIBRETRO_DEPS_DIR)/libchdr
-ZLIB_DIR             = $(LIBCHDR_DIR)/deps/zlib-1.2.11
 MINIZIP_DIR          = $(LIBRETRO_DEPS_DIR)/minizip
 LZMA_DIR             = $(LIBCHDR_DIR)/deps/lzma-19.00
 LIBRETRO_COMM_DIR    = $(LIBRETRO_DEPS_DIR)/libretro-common
 
+VORBIS_INCS := $(shell pkg-config --cflags vorbisfile)
+VORBIS_LIBS := $(shell pkg-config --libs vorbisfile)
+OGG_INCS := $(shell pkg-config --cflags ogg)
+OGG_LIBS := $(shell pkg-config --libs ogg)
+ZLIB_INCS := $(shell pkg-config --cflags zlib)
+ZLIB_LIBS := $(shell pkg-config --libs zlib)
+
 INCFLAGS :=  \
 	-I$(CORE_DIR)/src \
 	-I$(LIBRETRO_COMM_DIR)/include \
 	-I$(LIBRETRO_DEPS_DIR)/ \
 	-I$(LIBCHDR_DIR)/include \
 	-I$(LZMA_DIR)/include \
-	-I$(LIBOGG_DIR)/include \
-	-I$(LIBVORBIS_DIR)/include \
-	-I$(LIBVORBIS_DIR)/lib \
-	-I$(ZLIB_DIR) \
 	-I$(MINIZIP_DIR)
 
 SOURCES_CXX :=\
@@ -113,46 +113,17 @@
 	$(LZMA_DIR)/src/Sort.c
 
 SOURCES_C += \
-	$(LIBOGG_DIR)/src/bitwise.c \
-	$(LIBOGG_DIR)/src/framing.c
-
-SOURCES_C += \
-	$(LIBVORBIS_DIR)/lib/bitrate.c \
-	$(LIBVORBIS_DIR)/lib/block.c \
-	$(LIBVORBIS_DIR)/lib/codebook.c \
-	$(LIBVORBIS_DIR)/lib/envelope.c \
-	$(LIBVORBIS_DIR)/lib/floor0.c \
-	$(LIBVORBIS_DIR)/lib/floor1.c \
-	$(LIBVORBIS_DIR)/lib/info.c \
-	$(LIBVORBIS_DIR)/lib/lookup.c \
-	$(LIBVORBIS_DIR)/lib/lpc.c \
-	$(LIBVORBIS_DIR)/lib/lsp.c \
-	$(LIBVORBIS_DIR)/lib/mapping0.c \
-	$(LIBVORBIS_DIR)/lib/mdct.c \
-	$(LIBVORBIS_DIR)/lib/psy.c \
-	$(LIBVORBIS_DIR)/lib/registry.c \
-	$(LIBVORBIS_DIR)/lib/res0.c \
-	$(LIBVORBIS_DIR)/lib/sharedbook.c \
-	$(LIBVORBIS_DIR)/lib/smallft.c \
-	$(LIBVORBIS_DIR)/lib/synthesis.c \
-	$(LIBVORBIS_DIR)/lib/vorbisfile.c \
-	$(LIBVORBIS_DIR)/lib/window.c
-
-SOURCES_C += \
-	$(ZLIB_DIR)/adler32.c \
-	$(ZLIB_DIR)/crc32.c \
-	$(ZLIB_DIR)/deflate.c \
-	$(ZLIB_DIR)/infback.c \
-	$(ZLIB_DIR)/inffast.c \
-	$(ZLIB_DIR)/inflate.c \
-	$(ZLIB_DIR)/inftrees.c \
-	$(ZLIB_DIR)/trees.c \
-	$(ZLIB_DIR)/zutil.c
-
-SOURCES_C += \
 	$(MINIZIP_DIR)/ioapi.c \
 	$(MINIZIP_DIR)/unzip.c
 
 ifneq ($(STATIC_LINKING),1)
 SOURCES_C +=	$(SRC_LIBRETRO_COMMON)
 endif
+
+INCFLAGS += ${VORBIS_INCS}
+INCFLAGS += ${OGG_INCS}
+INCFLAGS += ${ZLIB_INCS}
+
+LIBS += ${VORBIS_LIBS}
+LIBS += ${OGG_LIBS}
+LIBS += ${ZLIB_LIBS}
